src/get_env.c

```diff
@@ -40,28 +35,13 @@ rcutils_get_env(const char * env_name, const char ** env_value)
   if (NULL == env_value) {
     return "argument env_value is null";
   }
-  *env_value = NULL;
-#ifdef _WIN32
-  size_t required_size;
-  errno_t ret = getenv_s(&required_size, __env_buffer, sizeof(__env_buffer), env_name);
-  switch (ret) {
-    case 0:
-      __env_buffer[WINDOWS_ENV_BUFFER_SIZE - 1] = '\0';
-      *env_value = __env_buffer;
-      break;
-    case EINVAL:
-      return "invalid arguments when reading environment variable";
-    case ERANGE:
-      return "insufficient buffer size to read environment variable";
-    default:
-      return "unknown error code reading environment variable";
-  }
-#else
+
+  // TODO(Suyash458): getenv is deprecated on Windows; consider using getenv_s instead
```

Comment:
@Suyash458 aren't we consciously ignoring that deprecation? Why keeping a TODO?

Reply:
I asked for that.  It's mostly so that we can make the even better (but bigger) change in step 2 in https://github.com/ros2/rcutils/pull/237#discussion_r408151034 .  At that point, we most likely will switch back to `getenv_s`.